aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -377,37 +377,58 @@ builder.On<string, string>("ReceiveMessage", (user, message) => ...
 await builder.StartAsync();
 ```
 
-### SignalR seamless reconnect
+### SignalR stateful reconnect
 
-SignalR seamless reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
 
-Seamless reconnect achieves this by:
+Stateful reconnect achieves this by:
 
 * Temporarily buffering data on the server and client.
 * Acknowledging messages received (ACK-ing) by both the server and client.
 * Recognizing when a connection is returning and replaying messages that may have been sent while the connection was down.
 
-To opt-in to seamless reconnect:
+To opt-in to stateful reconnect for a JavaScript or Typescript client:
 
-* Update the .NET client code to enable the `UseAcks` option:
+* Update the JavaScript or TypeScript client code to enable the `withStatefulReconnect` option:
+
+  ```JavaScript
+  const builder = new signalR.HubConnectionBuilder()
+    .withUrl("/hubname")
+    .withStatefulReconnect({ bufferSize: 1000 });  // Optional, defaults to 100,000
+  const connection = builder.build();
+  ```
+
+* The server also needs to enable support on the endpoint being accessed by the client:
+
+  ```csharp
+  app.MapHub<MyHub>("/hubName", options =>
+  {
+      options.AllowStatefulReconnects = true;
+  });
+  ```
+
+To opt-in to stateful reconnect for a .NET client:
+
+* Update the .NET client code to enable the `UseStatefulReconnect` option:
 
   ```csharp
   var hubConnection = new HubConnectionBuilder()
       .WithUrl("<hub url>",
                options =>
                {
-                  options.UseAcks = true;
+                  options.UseStatefulReconnect = true;
+                  options.StatefulReconnectBufferSize = 1000;  // Optional, defaults to 100,000
```

Comment:
I don't think this is how you enable it for the .NET client. I'll give a code snippet in a bit.

Reply:
@BrennanConroy I added the .NET Client example back in and updated based on what I was seeing in the SignalR code.  Could you verify if this is correct for the latest?

Reply:
```csharp
var builder = new HubConnectionBuilder()
      .WithUrl("<hub url>")
      .WithStatefulReconnect();
builder.Services.Configure<HubConnectionOptions>(o => o.StatefulReconnectBufferSize = 1000);
var hubConnection = builder.Build();
```

Reply:
@BrennanConroy, I added in your .NET client sample.  I also made the hub config step consistent for both examples, so good to give that a look over.  Thanks again for the help.